### PR TITLE
[REF] test_server.py: Create server configuration file and copy attachment

### PR DESCRIPTION
**Pull request by moylop260**
- Create server configuration file
   Usefully when you want start server manually without set --addons-path
 - Copy attachment from database template
 - Fix https://github.com/OCA/maintainer-quality-tools/issues/299

### CODE REVIEW

**Comment by oscarolar** [comment]
:+1:

File: travis/test_server.py [hunk 1]
```diff
@@ -276,7 +299,6 @@ def main(argv=None):
                      "-d", database,
                      "--stop-after-init",
                      "--log-level", test_loglevel,
-                     "--addons-path", addons_path,
```
Comment (dreispt) [inline travis/test_server.py]: Why not also pass the --data-dir option here, instead of using a conf file?

Comment (moylop260) [inline travis/test_server.py]: To re-use when start server normally without a param extra

**Comment by moylop260** [comment]
ping

**Comment by nhomar** [comment]
@moylop260 travis is red and coverage decrease, can you check that please?

**Comment by moylop260** [comment]
@nhomar 
Fixed https://github.com/vauxoo-dev/maintainer-quality-tools/commit/e59c9ea10680dd50be8f424d339360440b23f915
New version of flake8 inserted a new check, unrelated with this PR... but I fixed.

File: travis/test_server.py [hunk 2]
```diff
@@ -225,6 +225,25 @@ def run_from_env_var(env_name_startswith, environ):
         subprocess.call(command, shell=True)
 
 
+def create_server_conf(data, version):
```
Comment (osvalr) [inline travis/test_server.py]: version parameter is not used, why leave it here?

**Comment by osvalr** [comment]
:+1:

**Comment by nhomar** [comment]
:+1:

**Comment by moylop260** [comment]
Can be merged?

**Comment by moylop260** [comment]
Thank you @max3903

**Pull request merged**